images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -22,6 +22,11 @@ $markdown += New-MDList -Style Unordered -Lines @(
     "Image Version: $env:ImageVersion"
 )
 
+$markdown += New-MDHeader "Enabled windows optional features" -Level 2
+$markdown += New-MDList -Style Unordered -Lines @(
+    "Windows Subsystem for Linux"
```

Comment:
Can we determine and list all enabled features automatically somehow?

Reply:
Yes, but the list is huge and the titles are vague, this why i prefer to mention only the features we turn on explicitly